src/oscillator.js

```diff
@@ -236,11 +236,12 @@ define(function (require) {
       var now = p5sound.audiocontext.currentTime;
       var rampTime = rampTime || 0;
       var tFromNow = tFromNow || 0;
+      var t = now + tFromNow + rampTime;
       // var currentFreq = this.oscillator.frequency.value;
       // this.oscillator.frequency.cancelScheduledValues(now);
 
       if (rampTime === 0) {
-        this.oscillator.frequency.cancelScheduledValues(now);
+        // this.oscillator.frequency.cancelScheduledValues(now);
```

Comment:
commenting out this line solves an error with scheduling the changes of values. i think that we _don't_ want to 'cancelScheduledValues' because, doing results in losing every note except the last one scheduled.
ex. if we have a monosynth and
'monosynth.play(60,1,0,1);
monosynth.play(64,1,1,1);
monosynth.play(68,1,2,1);'
we will hear note 69 (A = 440) twice and note 68 (Ab = 415.3), because each value cancels all preceding values.